### DISCUSSION TITLE
Conditional tag with DAYS_TILL_START in emails?

### DISCUSSION THREAD

**Discussion by LicetBovi** [liedekef/events-made-easy#443] [Q&A] [answered]
Hi Franky,
First of all I would like to thank you for the great work you are doing with EME. It is really impressive to see all the effort you put into the plugin, especially compared to the developers of some paid plugins...

I hope my problem is easy to solve.
On a client site I need to send reminder emails to each person who has booked an event. Since the time between booking and the actual event date can be quite long, it would be desirable to send at least two reminders with texts that vary depending on the remaining days.
I have tried using a conditional tag like [eme_if tag='#ESC_DAYS_TILL_START' le 10] but it does not seem to work. I have tried several variations without success, while other conditional tags work as expected.
Perhaps I am missing something?

Any help would be greatly appreciated!
Thanks in advance!

**Comment by liedekef** [comment, marked as answer]
Your syntax seems wrong, try ` [eme_if tag='#ESC_DAYS_TILL_START' le='10']`.
Also: this only works for actual reminder mails, see the reminder setting:
![image](https://github.com/liedekef/events-made-easy/assets/75667/8e9f6303-6028-42fb-8e74-b6c52c2e4958)
and see the mail template for reminder mails.
This also works for own mails sent via the mail menu, but then only for mails of type "event mails" (not generic mails), since only for event mails the event placeholders are interpreted.

**Reply by LicetBovi** [reply]
Thank you for the prompt reply!
Obviously I missed the equal sign.

And yes - I want to use it in a reminder email.
